6-register/1-register.md

```diff
@@ -0,0 +1,40 @@
+# Register a Rococo Parachain
+
+Rococo is Parity's official test network for Cumulus-based parachains. The purpose of this document
+is to guide Cumulus parachain developers through the steps needed to register their parachain with
+the Rococo test network.
+
+## Write Your Parachain
+
+Follow the steps in the previous chapter to develop your parachain. In particular, refer to the
+[Parachain Template Overview](../5-develop/1-template-overview) and
+[Template Pallet](../5-develop/3-template-pallet) sections.
+
+## Test Your Parachain
+
+Follow the steps in the previous section to test your parachain. In particular, refer to the
+[Sending Messages](../5-develop/4-sending-messages) and
+[Receiving Messages](../5-develop/5-receiving-messages) sections.
```

Comment:
Here's an idea for an alternate introduction:

This page is supplemental material to the workshop for anyone who has built a parachain that they would like to register on the public testnet. Before attempting to register on the public network, you should work through the entire workshop, understand the material presented, and test your parachain locally.